com.getstation.Station.metainfo.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.getstation.Station</id>
+  <launchable type="desktop-id">com.getstation.Station.desktop</launchable>
+  <name>Station</name>
+  <project_license>Apache-2.0</project_license>
+  <developer_name>Station Team</developer_name>
+  <summary>One app to rule them all</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <url type="homepage">https://getstation.com</url>
+  <url type="bugtracker">https://github.com/getstation/desktop-app/issues</url>
+  <url type="help">https://github.com/getstation/desktop-app/wiki/FAQ</url>
+  <description>
+    <p>
+      Station was designed with a single purpose in mind: to improve your workflow, 
+      a goal that is met by changing the standard browser-based workspace. 
+      To be more specific, Station can place a variety of web applications under the 
+      same roof, making it easier for you to switch between the apps you use on a daily basis.
+    </p>
+    <p>
+      Station centralizes all your web tools into a single interface, eliminating 
+      the time needed to commute between different browser tabs or desktop apps. 
+      Having all the web apps you frequently use in a single place makes it easy 
+      for you to work with a large number of programs.    
+    </p>
+  </description>
+  <screenshots>
+    <screenshot type="default">
+      <image type="source">https://github.com/getstation/desktop-app/blob/main/docs/screenshots/1.png</image>
```

Comment:
you should never reference the tip of a branch because this is not immutable.

Reply:
also this is not a link to an image.

Reply:
> also this is not a link to an image.

true
fixed